src/experimentationService.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { addContextProperty, sendInfo } from "vscode-extension-telemetry-wrapper";
+import { getExperimentationService, IExperimentationService, IExperimentationTelemetry, TargetPopulation } from "vscode-tas-client";
+
+class ExperimentationTelemetry implements IExperimentationTelemetry {
+
+    public setSharedProperty(name: string, value: string): void {
+        addContextProperty(name, value);
+    }
+
+    public postEvent(eventName: string, props: Map<string, string>): void {
+        const payload: any = { __event_name__: eventName };
+        for (const [key, value] of props) {
+            payload[key] = value;
+        }
+
+        sendInfo("", payload);
+    }
+}
+
+let expService: IExperimentationService;
+
+export function getExpService() {
+    return expService;
+}
+
+export function initExpService(context: vscode.ExtensionContext): void {
+    const packageJson: {[key: string]: any} = require("../package.json");
```

Comment:
webpack can handle that. It will become something like

```
const packageJson = __webpack_require__(JSON.parse(<content of the package.json in string format>));

```